Libraries/LibWeb/CSS/Interpolation.cpp

```diff
@@ -82,19 +92,24 @@ static RefPtr<CSSStyleValue const> interpolate_scale(DOM::Element& element, Calc
     auto const& from_transform = from.as_transformation();
     auto const& to_transform = to.as_transformation();
 
-    auto interpolated_x = interpolate_value(element, calculation_context, from_transform.values()[0], to_transform.values()[0], delta);
-    auto interpolated_y = interpolate_value(element, calculation_context, from_transform.values()[1], to_transform.values()[1], delta);
-
+    auto interpolated_x = interpolate_value(element, calculation_context, from_transform.values()[0], to_transform.values()[0], delta, allow_discrete);
+    if (!interpolated_x)
+        return interpolate_discrete(from, to, delta, allow_discrete);
```

Comment:
Having to do this everywhere is definitely a bit awkward. Could `interpolate_value()` call `interpolate_discrete()` instead? I'm having trouble reasoning what the effects of that would be.

Maybe a separate function that wraps this behaviour if not.